build/CoreText/dll/CoreText.def

```diff
@@ -139,6 +139,7 @@ LIBRARY CoreText
         _CTFontGetDWriteWeight
         _CTFontGetDWriteStretch
         _CTFontGetDWriteStyle
+        _CTFontGetCompatibleFamilyName
```

Comment:
>_CTFontGetCompatibleFamilyName [](start = 8, length = 30)

why the two sets of apis? why not let this go directly through CG instead (just like before?)